web/src/app/Routes.tsx

```diff
@@ -17,72 +19,74 @@ import {
   LANDING_PAGE_ROUTE,
   MORE_CAST_2_ROUTE
 } from 'utils/constants'
-import MoreCastPage from 'features/fireWeather/pages/MoreCastPage'
+const MoreCastPage = lazy(() => import('features/fireWeather/pages/MoreCastPage'))
 import { NoMatchPage } from 'features/fireWeather/pages/NoMatchPage'
-import { FireBehaviourCalculator } from 'features/fbaCalculator/pages/FireBehaviourCalculatorPage'
-import { FireBehaviourAdvisoryPage } from 'features/fba/pages/FireBehaviourAdvisoryPage'
-import LandingPage from 'features/landingPage/pages/LandingPage'
-import MoreCast2Page from 'features/moreCast2/pages/MoreCast2Page'
+const FireBehaviourCalculator = lazy(() => import('features/fbaCalculator/pages/FireBehaviourCalculatorPage'))
+const FireBehaviourAdvisoryPage = lazy(() => import('features/fba/pages/FireBehaviourAdvisoryPage'))
+const LandingPage = lazy(() => import('features/landingPage/pages/LandingPage'))
+const MoreCast2Page = lazy(() => import('features/moreCast2/pages/MoreCast2Page'))
 import MoreCast2AuthWrapper from 'features/auth/components/MoreCast2AuthWrapper'
 
 const shouldShowDisclaimer = HIDE_DISCLAIMER === 'false' || HIDE_DISCLAIMER === undefined
 
 const WPSRoutes: React.FunctionComponent = () => {
   return (
     <Router>
-      <Routes>
-        <Route path={LANDING_PAGE_ROUTE} element={<LandingPage />} />
-        <Route
-          path={PERCENTILE_CALC_ROUTE}
-          element={<PercentileCalculatorPageWithDisclaimer showDisclaimer={shouldShowDisclaimer} />}
-        />
-        <Route path={FIRE_WEATHER_ROUTE} element={<Navigate to={MORECAST_ROUTE} />} />
-        <Route
-          path={MORECAST_ROUTE}
-          element={
-            <AuthWrapper>
-              <MoreCastPage />
-            </AuthWrapper>
-          }
-        />
-        <Route
-          path={HFI_CALC_ROUTE}
-          element={
-            <AuthWrapper>
-              <HfiCalculatorPage />
-            </AuthWrapper>
-          }
-        />
-        <Route path={C_HAINES_ROUTE} element={<CHainesPage />} />
+      <Suspense fallback={<div>Loading...</div>}>
```

Comment:
this is just placeholder. do you want use like LoadingBackdrop.tsx or something else?